tests/Unit/Domain/Creators/Test_CylindricalBinaryCompactObject.cpp

```diff
@@ -612,19 +502,162 @@ void test_parse_errors() {
           Options::Context{false, {}, 1, 1}),
       Catch::Matchers::Contains(
           "To use the CylindricalBBH domain with time-dependent maps"));
-  // Note: the boundary condition-related parse errors are checked in the
-  // test_connectivity function.
+  // Boundary condition errors
+  CHECK_THROWS_WITH(
+      domain::creators::CylindricalBinaryCompactObject(
+          {{2.0, 0.05, 0.0}}, {-5.0, 0.05, 0.0}, 1.0, 0.4, false, false, false,
+          25.0, false, 1_st, 3_st, create_inner_boundary_condition(),
+          std::make_unique<TestHelpers::domain::BoundaryConditions::
+                               TestPeriodicBoundaryCondition<3>>(),
+          Options::Context{false, {}, 1, 1}),
+      Catch::Matchers::Contains("Cannot have periodic boundary "
+                                "conditions with a binary domain"));
+  CHECK_THROWS_WITH(
+      domain::creators::CylindricalBinaryCompactObject(
+          {{2.0, 0.05, 0.0}}, {-5.0, 0.05, 0.0}, 1.0, 0.4, false, false, false,
+          25.0, false, 1_st, 3_st,
+          std::make_unique<TestHelpers::domain::BoundaryConditions::
+                               TestPeriodicBoundaryCondition<3>>(),
+          create_outer_boundary_condition(), Options::Context{false, {}, 1, 1}),
+      Catch::Matchers::Contains("Cannot have periodic boundary "
+                                "conditions with a binary domain"));
+  CHECK_THROWS_WITH(
+      domain::creators::CylindricalBinaryCompactObject(
+          {{2.0, 0.05, 0.0}}, {-5.0, 0.05, 0.0}, 1.0, 0.4, false, false, false,
+          25.0, false, 1_st, 3_st, nullptr, create_outer_boundary_condition(),
+          Options::Context{false, {}, 1, 1}),
+      Catch::Matchers::Contains(
+          "Must specify either both inner and outer boundary "
+          "conditions or neither."));
+  CHECK_THROWS_WITH(
+      domain::creators::CylindricalBinaryCompactObject(
+          {{2.0, 0.05, 0.0}}, {-5.0, 0.05, 0.0}, 1.0, 0.4, false, false, false,
+          25.0, false, 1_st, 3_st, create_inner_boundary_condition(), nullptr,
+          Options::Context{false, {}, 1, 1}),
+      Catch::Matchers::Contains(
+          "Must specify either both inner and outer boundary "
+          "conditions or neither."));
+}
+
+void test_cylindrical_bbh() {
+  MAKE_GENERATOR(gen);
+
+  const std::vector<double> times_to_check{{1., 10.}};
```

Comment:
Haven't reviewed this PR at all, but I was looking at this test because it keeps timing out and determined that this "10" case is over 2/3 of the runtime because of all the integration in the FoTs.  Do we need to include this case, or can we change it to a smaller number?

Reply:
I can probably lower it down to like 2.3 or something

Reply:
On my machine that reduces the test runtime by about a factor of 2, so that sounds good.

Reply:
That was just on develop.  I assumed it would be similar here since this PR doesn't appear to have affected the total runtime much.

Reply:
Ok made the change